src/engine/Server.h

```diff
@@ -38,29 +38,28 @@ class Server {
  private:
   //! Initialize the server.
   void initialize(const string& indexBaseName, bool useText,
-                  bool usePatterns = true, bool usePatternTrick = true,
+                  bool usePatterns = true,
                   bool loadAllPermutations = true);
 
  public:
   //! First initialize the server. Then loop, wait for requests and trigger
   //! processing. This method never returns except when throwing an exception.
   void run(const string& indexBaseName, bool useText, bool usePatterns = true,
-           bool usePatternTrick = true, bool loadAllPermutations = true);
```

Comment:
The pattern trick bool is initialized in the constructor now instead of keeping it uninitialized and the initialized via indirection